test_runner/src/main/kotlin/ftl/args/ValidateAndroidArgs.kt

```diff
@@ -138,3 +141,18 @@ private fun AndroidArgs.assertRoboTest() {
 private fun AndroidArgs.assertOtherFiles() {
     otherFiles.forEach { (_, path) -> ArgsHelper.assertFileExists(path, "from otherFiles") }
 }
+
+private fun AndroidArgs.checkEnvironmentVariables() =
+    if (environmentVariables.isNotEmpty() && directoriesToPull.isEmpty())
+        println("WARNING: environment-variables set but directories-to-pull, not this can cause coverage file will not download to the bucket")
```

Comment:
```suggestion
        println("WARNING: environment-variables set but directories-to-pull is empty, this will result in the coverage file  not downloading to the bucket")
```